ui/imports/shared/panels/StatusAssetSelector.qml

```diff
@@ -41,6 +46,11 @@ Item {
         id: d
         property string iconSource: ""
         property string text: ""
+        property string searchString
+
+        readonly property var updateSearchText: Backpressure.debounce(root, 1000, function(inputText) {
```

Comment:
Why it is a property and not a regular function ? 

`function updateSearchText(inputText) {
        Backpressure.debounce(root, 1000, function(inputText) {
                    d.searchString = inputText
                })
  }`

Reply:
it could b either of those? is there a reason why it should be a function?